### ISSUE TITLE
Content changes automatically

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#197]
Screengrab/videos (insert file names here)
2020 11 07, Content changes automatically
[2020 11 07, Content changes automatically.zip](https://github.com/CityOfBoston/Iterators/files/5514410/2020.11.07.Content.changes.automatically.zip)
.mp4

Link
https://www.boston.gov/my-neighborhood

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.7 > Safari

Trusted Tester Test
WCAG 2.4.3:  https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html 

Brief Description
Content changes automatically

Steps to reproduce
1)	00:00, Open the link in a private browser window
a)	Note:  an address is already present in the search field.
2)	00:02, Use keyboard navigation to select the search result and press enter to show the neighborhood information.
3)	00:07, Continue to use keyboard navigation to tab to different information areas.
4)	00:08, Start pressing the shift-tab keys to navigate backwards to the search field.
5)	00:09, Use Shift-Tab to navigate back to the search field.
a)	Bug:  Note that the search results disappear and are replaced with a list containing the single address and the neighborhood information disappears.  The user must then reselect the address in order for the neighborhood information to reappear (not shown)

Desired Result
•	A change in web content does not occur when an element receives Focus.  Accessibility users may navigate through the interface with a keyboard which will give different elements focus as they are encountered.
•	Suggestion is to change this so that the search is updated if a user types in the search field, indicating that a search is in progress, but not when receiving focus through keyboard navigation.

Actual Result
•	The page is updated when the keyboard is used to navigate into the search field.

**Comment by reillyzlab** [comment]
Agree with this suggestion. 

This can be/should be implemented by a developer.

**Comment by subaha-cob** [comment]
@david-iterators  UAT is now up with MNL tickets. 
https://d8-uat.boston.gov/

Try testing with this addresses only:

50 Milk St #2050 Boston, 02108
192 Dudley St Roxbury, 02119
27 Tyler St Boston, 02111
1 State St Ste 1500 Boston, 02108

**Comment by subaha-cob** [comment]
@jill-iterators can you also take a look at this. Thanks!

**Comment by jill-iterators** [comment]
Hi Stella,

These are the Jira tickets we looked at today and #197 is included. The only issue left to test is that #197 has a new bug – which was reported to Git Hub today and is #203. Bug report #197 is blocked by #208 so this needs to be fixed and then we’ll be happy to verify. Everything else is fixed and closed.

Let me know if you have any questions. If I don’t hear back from you, stay safe, and have a nice holiday.


-- Jill
Iterators LLC
Certified – WBENC, WOSB, WBE, Trusted Tester – DHS
617-909-0564 (cell)
jwillcox@IteratorsTesting.com<mailto:jwillcox@IteratorsTesting.com>
[signature_252065731]

**Comment by subaha-cob** [comment]
@jill-iterators issue #197 is a separate issue from issue #203 and #208. Can you test only for the functionality you list above. UAT does not have content for the entire addresses that is why I specifically ask to test ONLY the following addresses:

50 Milk St #2050 Boston, 02108
192 Dudley St Roxbury, 02119
27 Tyler St Boston, 02111
1 State St Ste 1500 Boston, 02108

**Comment by jill-iterators** [comment]
Hi Stella,
Sorry about that.  I read 203 as 208, a like formation.

The issue between #197 is due to the functionality, that is supposed to appear after searching for a certain area and does not show up on the page after the search happens. What happens after the search button is pressed is that the Search text field and the search button disappear and the neighborhood information is not shown. Let me know if you have further questions.

-- Jill
Iterators LLC
Certified – WBENC, WOSB, WBE, Trusted Tester – DHS
617-909-0564 (cell)
jwillcox@IteratorsTesting.com<mailto:jwillcox@IteratorsTesting.com>
[signature_2096184077]

**Comment by subaha-cob** [comment]
I am a little confuse @jill-iterators Are you searching only the addresses below?

50 Milk St #2050 Boston, 02108
192 Dudley St Roxbury, 02119
27 Tyler St Boston, 02111
1 State St Ste 1500 Boston, 02108

Any address outside the specific ones above will fail and give you an empty string. Clicking any other address field other than the ones mentioned here will do exactly what you said because the content is not there.

If I am still not getting what you are saying, please feel free to add a video for me to see the bug.

**Comment by jill-iterators** [comment]
Hi Stella, The testers point is that once you select the search result as described in #208, you cannot run the sequence in #197. We agree that #197 is fixed and can be closed and that #208 is a new outstanding bug.  Take good care, stay safe.

-- Jill
Iterators LLC
Certified – WBENC, WOSB, WBE, Trusted Tester – DHS
617-909-0564 (cell)
jwillcox@IteratorsTesting.com<mailto:jwillcox@IteratorsTesting.com>
[signature_1648015611]
